dll/cpl/mmsys/lang/fr-FR.rc

```diff
@@ -234,13 +234,13 @@ BEGIN
     IDS_CPLDESCRIPTION "Change le modèle de sons de votre ordinateur, ou configure les paramètres de vos hauts-parleurs et de vos périphériques d'enregistrement."
     IDS_NO_SOUND "(Aucun)"
     IDS_NO_DEVICES "Aucun périphérique"
-    IDS_SPEAKER_VOLUME "Speaker Volume"
-    IDS_SPEAKER_LEFT "&Left:"
-    IDS_SPEAKER_RIGHT "&Right:"
-    IDS_SPEAKER_CENTER "&Center:"
-    IDS_SPEAKER_BACK_LEFT "&Back left:"
-    IDS_SPEAKER_BACK_RIGHT "B&ack right:"
-    IDS_SPEAKER_LOW_FREQUENCY "Low &Frequency:"
+    IDS_SPEAKER_VOLUME "Volume des enceintes"
+    IDS_SPEAKER_LEFT "&Gauche :"
+    IDS_SPEAKER_RIGHT "&Droite :"
+    IDS_SPEAKER_CENTER "&Centre :"
```

Comment:
```suggestion
    IDS_SPEAKER_CENTER "&Centrale :"
```

Reply:
Hmmmm...

Reply:
linked with "enceinte".
Suggestion from Serge makes sense to me, but "centre" was not wrong either. I let you make you mind, override if required and merge :-)